docs/.vitepress/theme/custom.css

```diff
@@ -1,40 +1,36 @@
 :root {
-  --vp-c-purple-1: #7135be;
-  --vp-c-purple-2: #7f42cd;
-  --vp-c-purple-3: #9555e2;
-  --vp-c-purple-soft: rgba(155, 91, 233, 0.14);
   --vp-c-red: #f43f5e;
   --vp-c-green: #10b981;
   --vp-c-blue: #0092ff;
   --vp-c-purple: #a463f2;
-  --vp-c-brand-1: var(--vp-c-purple-1);
-  --vp-c-brand-2: var(--vp-c-purple-2);
-  --vp-c-brand-3: var(--vp-c-purple-3);
-  --vp-c-brand-soft: var(--vp-c-purple-soft);
-  --vp-font-family-base: -apple-system, BlinkMacSystemFont, "avenir next", avenir, helvetica, "helvetica neue", ubuntu,
-    roboto, noto, "segoe ui", arial, sans-serif;
-  --sans-text: Inter, var(--vp-font-family-base);
+  --theme-phosphate: #148576;
+  --theme-phosphate-soft: #d7fbf7;
+  --vp-c-brand-1: var(--theme-phosphate); /* link and brand color */
+  --vp-c-brand-2: var(--theme-phosphate);
+  --vp-c-brand-3: rgb(243, 139, 233); /* home page alt color */
```

Comment:
This doesn’t seem right: these three colors should have the same hue, but varying lightness.

Look at how the current button varies in color as you hover and click:

https://github.com/user-attachments/assets/f36bd40b-cf99-4f86-82d2-6dd07c880baf

Compare that to this branch, where the color changes from pink to green on hover, and there’s no variation in color on click:

https://github.com/user-attachments/assets/8eaf7830-7938-4e87-8646-fd5ae9280a29

Reply:
Oops! that's right. Since I couldn't find documentation, I went back to the defaults. As you're saying brand 1, 2, 3 are defined as colors of increasing lightness in light mode, and increasing darkness in dark mode.

![brand-light](https://github.com/user-attachments/assets/8464551c-40d4-48f5-b7d5-5c7b0a44d15c)

![brand-dark](https://github.com/user-attachments/assets/655b5257-1d0d-472d-a15a-a17c5e94c961)

I've done the same by deriving from phosphate in both modes, and re-adding the contrasting ("pink") color under another name.